jit-binding-server/src/main/kotlin/io/github/typesafegithub/workflows/jitbindingserver/ActionCoords.kt

```diff
@@ -0,0 +1,17 @@
+package io.github.typesafegithub.workflows.jitbindingserver
+
+import io.github.typesafegithub.workflows.actionbindinggenerator.domain.ActionCoords
+import io.ktor.http.Parameters
+
+fun ActionCoords(
+    parameters: Parameters,
+    extractVersion: Boolean,
+): ActionCoords {
```

Comment:
How about:

```suggestion
fun Parameters.extractActionCoords(
    extractVersion: Boolean,
): ActionCoords {
```
? The constructor-like function looks a bit weird to me.